Community/iot_device_registration/README.md

```diff
@@ -0,0 +1,97 @@
+# IoT Device Registration
```

Comment:
Hi Muhammed, the README looks good but I ask that you add the copyright date and the Apache license info at the top of the page for consistency with all other READMEs. Please have a look at the other README files in workflows in the Community folder.

Reply:
README updated